python/nav/models/api.py

```diff
@@ -59,7 +59,7 @@ class APIToken(models.Model):
     )
 
     def __str__(self):
-        return self.token
+        return f"token no. {self.id}"
```

Comment:
Assuming an `from django.utils.text import Truncator`, how about:


```suggestion
        descr = f" ({Truncator(self.comment).chars(25)})" if self.comment else ""
        return f"token #{self.id}{descr}"
```
?